admin_guide/cluster_management.rst

```diff
@@ -127,3 +127,45 @@ place for the master node, users can dynamically reconstruct this metadata from
 shard information available on the worker nodes. To learn more about the metadata
 tables and their schema, you can visit the :ref:`metadata_tables` section of our documentation.
 
+Tenant Isolation
+################
+
+Citus places table rows into worker shards based on the hashed value of the rows' distribution column.  Multiple distribution column values often fall into the same shard. In the Citus multi-tenant use case this means that tenants often share shards.
```

Comment:
Agreed to leave it out. We're not emphasizing append, so its OK to just skip that.